tooltips.js

```diff
@@ -0,0 +1,27 @@
+
+function setChildVisibility(element, visibility) {
+    const child = element.firstElementChild
+    if (child != null)
+        child.style.visibility = visibility
+}
+
+const clickButton = document.getElementById("click-button")
+clickButton.onclick = () => {
+    setChildVisibility(clickButton, "visible")
+    setTimeout(() => {
```

Comment:
Why timeout? What if will click this button many times?

In case you want add timer, you can get timer id as return value of `setTimeout` and pass it into `clearTimeout` to disable it on second click